server/api/users.js

```diff
@@ -53,3 +53,68 @@ router.get('/:id/latestguesses', async (req, res, next) => {
 		next(error)
 	}
 })
+
+const getRandomInt = max => {
```

Comment:
I also think we should move all wordle related routes to a wordleGames.js file in the api, and change the api route from being /users to being /wordle. I think this would help with the cleanliness of our code!